docs_user/modules/proc_updating-the-object-storage-endpoints.adoc

```diff
@@ -54,7 +55,7 @@ DEBUG:swiftclient:RESP BODY: b'[]'
 . Run tempest tests against object-storage:
```

Comment:
Can we put here an if so we can list this verification step (tempest) only upstream? We're not really asking our customers to run tempest, so I would say that it's best to keep it upstream only.

Reply:
Yes, I made the step upstream only.